config/development.toml

```
@@ -900,6 +900,20 @@ payout_analytics_topic = "hyperswitch-payout-events"
 consolidated_events_topic = "hyperswitch-consolidated-events"
 authentication_analytics_topic = "hyperswitch-authentication-events"
 
+[debit_routing_config]
+fraud_check_fee = "0.01"
```

Is it possible to have these values as numbers in the config? With strings, there's a chance of a non-numeric value sneaking in somewhere.
```suggestion
[debit_routing_config]
fraud_check_fee = 0.01
```